src/DependencyInjection/PmgPheanstalkExtension.php

```diff
@@ -72,4 +76,27 @@ private static function serviceName($name)
     {
         return sprintf('pmg_pheanstalk.%s', $name);
     }
+
+    /**
+     * Loads all internal service definition files into the container
+     */
+    private function loadFiles(array $config, ContainerBuilder $container)
```

Comment:
I would not load files here, just add the definitions in PHP above. There's no need to load a file for a single service.